src/Functions/grouping.h

```diff
@@ -19,10 +19,15 @@ class FunctionGroupingBase : public IFunction
     static constexpr UInt64 ONE = 1;
 
     const ColumnNumbers arguments_indexes;
+    const bool force_compatibility;
```

Comment:
in context of this pr it is clear what `compatibility` you're referring to, but for the future reader it probably will be not easy to get. add pls a brief comment